activiti-cloud-service-common/activiti-cloud-service-messaging-config/src/main/java/org/activiti/cloud/common/messaging/functional/Connector.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.common.messaging.functional;
+
+import java.util.function.Function;
+
+@FunctionalInterface
+public interface Connector<T, R> extends Function<T, R> {
```

Comment:
Sorry for the late thought, but wouldn't be better to rename this interface to `CloudConnector` to differentiate from the existing [Connector](https://github.com/Activiti/Activiti/blob/develop/activiti-api/activiti-api-process-runtime/src/main/java/org/activiti/api/process/runtime/connector/Connector.java) interface?